src/services/userToRoomJoiner.js

```diff
@@ -0,0 +1,61 @@
+const { User, Room } = require('../entities')
+
+class UserToRoomJoiner {
```

Comment:
Generally why services are all classes?
Why can't they be modules, I mean they only have methods inside.

Reply:
They can, i just prefer this way. But this will be a function which returns another function to support dependencies injection (repos, for instance)
Also, you may want to pass additional parameters in constructor, to not pass it in every function ( possible in case of function returning a function), you may want to extend functionality via inheritance